src/brainfuck.c

```diff
@@ -81,28 +82,31 @@ static int cfbf_generate_jumps(cfbf_state *state)
                                 state->commands[i].jmp_ptr = (int32_t)jmp_fwrd;
                         } else {
                                 fprintf(stderr, "Unopened close bracket found!\n");
-                                return 1;
+                                goto err;
```

Comment:
`85, 92, 98-100` fixes the potential leak when encountering unbalanced brackets.

Before it was just returning without thinking about destroying the stack which could still contain data.